runtime/common/src/apis.rs

```diff
@@ -1124,6 +1124,10 @@ macro_rules! impl_runtime_apis_plus_common {
 						hex_literal::hex!(  "0d715f2646c8f85767b5d2764bb27826"
 											"04a74d81251e398fd8a0a4d55023bb3f")
 							.to_vec().into(),
+						// Parameters Parameters
```

Comment:
Parameters Parameters is a map, not a value.
This whitelist do nothing then, as we can whitelist only full keys

Reply:
Is there no option to whitelist an entire storage?

Reply:
I don't think so, we have to edit manually this list each time we add a parameter

Reply:
fixed